src/Views/OperatingSystemView.vala

```diff
@@ -183,12 +188,24 @@ public class About.OperatingSystemView : Gtk.Box {
             xalign = 0
         };
 
+        update_progress_bar = new Gtk.LevelBar.for_interval (0, 100) {
```

Comment:
This should be a progress bar since we are indicating progress rather than level :)